platform-sdk/swirlds-common/src/testFixtures/java/com/swirlds/common/test/fixtures/Randotron.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.common.test.fixtures;
+
+import com.hedera.pbj.runtime.io.buffer.Bytes;
+import com.swirlds.common.crypto.DigestType;
+import com.swirlds.common.crypto.Hash;
+import com.swirlds.common.crypto.Signature;
+import com.swirlds.common.crypto.SignatureType;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.time.Instant;
+import java.util.Random;
+
+/**
+ * An extension of the Random class that provides additional utility methods for generating random data.
+ */
+public class Randotron extends Random {
```

Comment:
Maybe also final since we need to limit the mechanisms that allow the creation of instances of this without the seed

Reply:
Good point - done